addons/udes_stock_cron/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "UDES Stock Cron",
+    "summary": "udes stock cron",
+    "description": "Core models and configuration for UDES Stock cron - Odoo 14",
+    "author": "Unipart digital",
+    "website": "http://github/unipartdigital/udes-open",
+    "category": "UDES",
+    "version": "0.1",
+    "depends": ["base", "stock", "stock_picking_batch", "udes_common", "udes_cron", "udes_stock"],
+    "data": [
+        "data/ir_cron.xml",
+        "views/stock_picking_type.xml",
+    ],
+}
```

Comment:
```suggestion
}

```
Nit: new line at the end of file